store/mongo/datastore_mongo.go

```diff
@@ -218,7 +228,7 @@ func (db *DataStoreMongo) InsertWorkflows(ctx context.Context, workflows ...mode
 			query := bson.M{"_id": workflow.Name}
 			update := bson.M{"$set": workflow}
 			if _, err := collWflows.UpdateOne(ctx, query, update, opt); err != nil {
-				return i, err
+				return i + 1, err
```

Comment:
Why are you returning + 1? I'm not 100% sure, but the idea here was to return the number of workflows which got successfully inserted into the database.